### ISSUE TITLE
Upload one's corpus

### ISSUE THREAD

**Issue by tommv** [nbonfils/bibliograph2#5]
Beside querying OA through its interface, Bibliograph's should allow the users to upload exixting OpenAlex corpus:
- previously created through Bibliograph
- or obtained otherwise

**Comment by nbonfils** [comment]
Yes this is a very good idea, first we would need to decide which format this user uploaded data should have, this is discussed in #2.
